WorkflowConcurrency/Tests/WorkerTests.swift

```diff
@@ -38,8 +38,120 @@ class WorkerTests: XCTestCase {
         disposable?.dispose()
     }
 
+    func testWorkflowUpdate() {
+        // Create the workflow which causes the TaskTestWorker to run.
+        let host = WorkflowHost(
```

Comment:
to clarify: is this expected to enter into a rendering loop given the behavior of the workflow/worker?

Reply:
The init for WorkflowHost calls render so it's expected that the render loop start the worker after the host is created.